config/matrix-api-v1.md

```diff
@@ -0,0 +1,193 @@
+# Matrix Service API - Version 1
+
+## Background
+
+The HCA matrix service allows users to request expression data from the HCA and receive an
+expression matrix along with associated metadata in the format of their choice.
+
+The first version of the matrix service's API focused on integration with the HCA Data Browser. So,
+it required that requests for expression data be specified using "bundle ids". "Bundles" are a
+concept internal to the HCA's data storage system. Since the data browser is closely integrated with
+the storage system, bundle ids were a natural way to specify requests. However, bundles ids make
+less sense for external clients. The focus of this API release is to enable requests for expression
+data specified in terms preferred by a scientist who is unfamiliar with the internals of the HCA's
+data platform.
+
```

Comment:
The Data Browser team (+ @briandoconnor) request that we retain the "URL to a manifest" API long term and in the future add a "reference to a Collection ID" API. These two interfaces allow the Data Browser to pass specific versioned bundles/files to the Matrix Service so that users can reproduce results from e.g. cited data or personal collections or data releases. These APIs should be in addition to this "filter query" interface.